src/Network/Network/help/New-AzFirewallThreatIntelWhitelist.md

```diff
@@ -0,0 +1,129 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azfirewallthreatintelwhitelist
+schema: 2.0.0
+---
+
+# New-AzFirewallThreatIntelWhitelist
+
+## SYNOPSIS
+Create a new threat intelligence whitelist for Azure Firewall
+
+## SYNTAX
+
+```
+New-AzFirewallThreatIntelWhitelist [-FQDN <String[]>] [-IpAddress <String[]>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzFirewallThreatIntelWhitelist** cmdlet creates a threat intel whitelist for Azure Firewall.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-AzFirewallThreatIntelWhitelist -IpAddress @("2.2.2.2", "3.3.3.3") -FQDN @("bing.com", "yammer.com")
+```
+
+This example creates a threat intel whitelist containing a FQDN whitelist of two entries and an Ip address whitelist of two entries
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FQDN
+The FQDNs of the Threat Intel Whitelist
+
+```yaml
+Type: System.String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IpAddress
+The IP Addresses of the Threat Intel Whitelist
+
+```yaml
+Type: System.String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
```

Comment:
After you remove `SupportsShouldProcess = true` and re-generate the doc. `-Confirm` and `-WhatIf` should be gone